lib/docs/wasm/main.zig

```diff
@@ -938,6 +938,17 @@ fn file_source_html(
 
     var cursor: usize = token_starts[start_token];
 
+    var indent: usize = 0;
+    if (std.mem.lastIndexOfAny(u8, ast.source[0..cursor], "\r\n")) |newline_index| {
```

Comment:
Canonical Zig style uses LF line endings (`\n`). Using `\n` here is also conveniently compatible with `\r\n`, so replacing this with `"\n"` will handle both line endings in common use today.